src/components/Registration/BusinessNumber.vue

```diff
@@ -29,8 +29,8 @@
         </p>
         <HelpBusinessNumber
           class="mt-4"
-          :isTypePartnership="isTypePartnership"
-          :isTypeSoleProp="isTypeSoleProp"
+          :isEntityPartnership="isEntityPartnership"
+          :isEntitySoleProp="isEntitySoleProp"
```

Comment:
This is where global-search-and-replace can trip you up, but I renamed the props in sub-component HelpBusinessNumber (which is only used here so no external impact).